src/main/java/seedu/duke/Duke.java

```diff
@@ -45,6 +49,50 @@ private static void quickStart() {
         System.out.println(newProfile);
     }
 
+    /**
+     * Prints the main menu of the application
+     * and requests for command.
+     *
+     */
+    private static void getCommand() {
+        // Request for first command
+        System.out.println("What would you like to do today?");
+        Scanner input = new Scanner(System.in);
+
+        while (true) {
+            ui.showMainMenu();
+
+            String fullCommand = input.nextLine();
+            String[] fullCommandSplit = fullCommand.split(" ", 2);
+
+            String description = "";
+            String command = fullCommandSplit[0];
+            if (fullCommandSplit.length > 1) {
+                description = fullCommandSplit[1];
+            }
+
+            if (command.equals("addprofile")) {
+                addProfile(description);
+            } else if (command.equals("editprofile")) {
+                editProfile(description);
+            } else if (command.equals("browse")) {
+                browseAnime(description);
+            } else if (command.equals("watchlist")) {
+                createWatchlist(description);
+            } else if (command.equals("add")) {
+                addToWatchlist(description);
+            } else if (command.equals("bookmark")) {
+                bookmarkAnime(description);
+            } else if (command.equals("help")) {
+                showHelp();
+            } else if (command.equals("exit")) {
+                return;
+            } else {
+                ui.showInvalidCommand();
+            }
```

Comment:
Perhaps this could be replaced with a switch statement if you feel it would be neater.